src/main/java/calculator/validator/InputValidator.java

```diff
@@ -0,0 +1,73 @@
+package calculator.validator;
+
+import calculator.constant.Constant;
+
+public class InputValidator {
+
+    private static final String INPUT_ERROR = "입력값이 올바르지 않습니다.";
+    private static final String OUT_OF_RANGE_ERROR = "int 범위를 초과합니다.";
+
+    public static void validateInput(String inputString) {
+        containNegative(inputString);
+        validateCustomFormat(inputString);
+    }
+
+    private static void containNegative(String inputString) {
+        if (!inputString.startsWith(Constant.NEGATIVE_CUSTOM_DELIMITER)) {
+            if (inputString.contains(Constant.NEGATIVE_SIGN)) {
+                throw new IllegalArgumentException(INPUT_ERROR);
+            }
+        }
+    }
+
+    private static void validateCustomFormat(String inputString) {
+        if (inputString.startsWith(Constant.CUSTOM_DELIMITER_PREFIX)) {
+            if (!inputString.substring(Constant.CUSTOM_DELIMITER_SUFFIX_START, Constant.CUSTOM_DELIMITER_SUFFIX_END)
+                    .equals(Constant.CUSTOM_DELIMITER_SUFFIX)) {
+                throw new IllegalArgumentException(INPUT_ERROR);
+            }
+        }
+    }
+
+    public static void validateStringOperand(String[] operands) {
+        validateBlank(operands);
+        validateTypeAndBound(operands);
+        validateResultBound(operands);
+    }
+
+    private static void validateBlank(String[] operands) {
+        if (operands.length > 1) {
+            for (String operand : operands) {
+                if (operand.equals(Constant.BLANK)) {
```

Comment:
문자열은 isBlank()라는 메소드를 사용할 수 있답니다😃
```suggestion
                if (operand.isBlank()) {
```

Reply:
기본 메서드에 대해 더욱 공부해야겠습니다!!